components/Form.js

```diff
@@ -158,10 +154,10 @@ export default function BillForm({ onSubmit, bill, isEditPage }) {
 const Wrapper = styled.form`
   display: flex;
   flex-direction: column;
```

Comment:
`display: grid`does the same as `display: flex` and `flex-direction` together. :D